contents/guides/users/different_streams.md

```diff
@@ -0,0 +1,67 @@
+
+The *Activity stream* and its filtered versions are the only streams that aren't sorted chronologically. Instead, the posts in this stream are sorted by their latest interaction.
+
+### Can I restrict the posts in my stream to just those that I was mentioned in?
+
+Yes. To show only those posts that other people thought to be especially relevant for you so they mentioned you in their posts, click “@Mentions” in the left-hand column on your main Stream page. Here uou can find all posts in which you've been mentioned. You can read more about interactions with people at “[How do I interact with people?][interact_with_people]”  
+The posts shown in your *@Mentions Stream* are sorted chronologically, with the newest post at the top.
+
+### Can I restrict the posts in my stream to just those from certain aspects?
+
+Yes. This stream is usually called the *Aspects Stream*. Click “My aspects” in the left side-bar and then click individual aspects in the list to select or deselect them. Only the posts by people in the selected aspects will appear in your stream. You can limit the selection of aspects to be displayed to as few or many as your aspects as you want. Once you've selected a set of aspects, these will be saved for this session and device. You can read more about aspects at “[What are aspects and how do I use them?][aspects]”  
+The posts shown in your *Aspects Stream* are sorted chronologically, with the newest post at the top.
+
+### Can I restrict the posts in my stream to just those with a certain tag?
+
+Yes. This stream is usually called the *Tags Stream*. Click “#Followed tags” in the left side-bar and your entire list of followed tags will expand. In your tags stream, you will see all public posts that contain at least one of your followed tags. To only display the stream for one tag, click that tag in the side-bar or in a post. You can read more about tags at “[How do I view and find content?][view_and_find_content]”    
+The posts shown in your *#Followed tags Stream* are sorted chronologically, with the newest post at the top.
+
+### Can I see all posts that are available on my pod?
+
+Yes, this is called the *Public Stream*. Click “Public activity” in the left side-bar to display all public posts that your pod knows about. Public streams may look different on different pods, some containing different, more, or less posts than those on other pods. This is due to diaspora\*'s decentralized nature. Each pod only knows about those posts, that have been sent to this pod. This means the public stream of your pod contains public posts from all local users, their contacts, and posts that have been reshared by their contacts.  
+The posts shown in the *Public activity Stream* are sorted chronologically, with the newest post at the top.
+
+## How can I navigate through my stream by using keyboard shortcuts?
+
+In the stream view you can use the following keyboard shortcuts:
+
+* j – Jump to the next post
```

Comment:
Can we put the keys themselves into backticks (`), so they get rendered with a bit of visual difference?